ex7/r_sakaki/gmm.py

```diff
@@ -0,0 +1,255 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import csv
+import argparse
+import os
+from matplotlib import cm
+from matplotlib.animation import FuncAnimation as animation
+
+
+def load_file(path):
+    with open(path) as f:
```

Comment:
```suggestion
def load_file(path: str) -> np.ndarray:
    """
    csvファイルを読み込んでnumpy配列で返す関数
    
    Args:
        path(str): 読み込むファイルのパス
    Returns:
        np.darray: 読み込んだデータのnumpy配列
    """
    with open(path) as f:
```
このように，関数やクラスにはDocstringと型アノテーションをつけるといいです．
Docstringは， `def` の次の行の"""で囲まれた部分で，
- 概要
- 引数
- 返り値
を書きます．

型アノテーションは， `def`の行に引数に対しては`引数名: 型`の形で，返り値に対しては `def func_name() -> 型:` の形で書きます．

どちらも，プログラムの可読性の向上に役立ったり，VSCodeでは関数の呼び出し場所でマウスオーバーするとDocstringが表示されたり，補完が効くようになったりして便利です．
また，型アノテーションを書いた状態で `def` の次の行で"""と入力するとある程度Docstringを自動で書いてくれるVSCode拡張の[autoDocstring - Python Docstring Generator](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring)があります．

今後も役に立つと思うので，main以外の関数について，Docstringを書いてみてください．できれば型アノテーションもあるとbetterです．